### DISCUSSION TITLE
Does Crimson Work for chromeOS 130?

### DISCUSSION THREAD

**Discussion by Charlieis1nsan3** [3kh0/ext-remover#1336] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I was pretty confused, but in SP-Scripts chromebook-utilities thing, in the exploit thing their was an exploit called crimson, which basically allowed the user to corrupt, extensions. Does this work in v130?

**Comment by v4pl** [comment]
Um crimson never heard can u send guide? And u can just test yourself.

**Reply by Charlieis1nsan3** [reply]
yeah but my mom is strict as fuck, so when I power wash, idk the password lol

**Reply by Charlieis1nsan3** [reply]
https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/%20Corrupt%20Extensions%20-%20Crimson

**Reply by S-PScripts** [reply]
> yeah but my mom is strict as fuck, so when I power wash, idk the password lol

it doesn't seem like any one is willing to test here, so you don't really have any other choices available for figuring out if crimson works for v130+. maybe you can join the TN discord .gg/unblock and see if there are any updates to the crimson kajig. (I cannot join myself maybe ask someone else to find it if you can't join too)

**Reply by Charlieis1nsan3** [reply]
S-PScirpts I already tested it doesn't work. It delets it but then it randomly comes back somehow

**Reply by SlitheringLizard1** [reply]
⠀
